tools/ci/appveyor/appveyor_build.sh

```diff
@@ -93,7 +93,7 @@ function set_configuration {
 #    run "Content of $FILE" cat Makefile.config
 }
 
-PARALLEL_URL='https://ftpmirror.gnu.org/parallel/parallel-latest.tar.bz2'
+PARALLEL_URL='https://git.savannah.gnu.org/cgit/parallel.git/tree/src/parallel'
```

Comment:
```suggestion
PARALLEL_URL='https://git.savannah.gnu.org/cgit/parallel.git/plain/src/parallel'
```

Reply:
this url returned html for me. With this change it should return plain text

Reply:
Oops - hopefully corrected version pushed, thanks!